### ISSUE TITLE
Include enable function in OmniFaces.Unload

### ISSUE THREAD

**Issue by hrkistiansen** [omnifaces/omnifaces#326]
It would be nice if there was some hook to re-enable the unload-event, even after a non-ajax submit.

We have a use case where a submit action returns a dynamic link which opens in a new window. The unload-event on the origin page is disabled by the submit action, and we have found no obvious way to aviod this behaviour, or re-enable unload-event.

Regards,
Håvard

**Comment by BalusC** [comment]
Reproducer:

```
<h:form target="_blank">
	<h:commandButton action="#{omniViewScopedBean.submit}" />
</h:form>
```

**Comment by BalusC** [comment]
This is a nasty corner case. Another issue is that the view scoped bean of parent page gets unloaded when child window is closed while parent window is still open.

**Comment by hrkistiansen** [comment]
Thank you for the update. The corner is even darker than you think.

In IE 10/11 the browser treats `window.open` and `<a target="_blank"` differently than `<form target="_blank"`. But _only_ if the parent window was opened by another window, _and_ the user has the setting "always open pop-ups in new window", _and_ the originating page uses PrimeFaces `<p:poll`. Using `<form target="_blank"` is the only way we have found to prevent the pop-up window to hide behind the originating window in this rather narrow corner case.

Of course, all is fine if the parent window is opened directly by the user, _or_ the browser setting is anything but "always open pop-ups in new window", _or_ parent page does not use a `<p:poll`, _or_ using any other browser.

We tried any number of ways to remedy this window-behaviour, but in the end settled for submit, re-enabling the unload-event, and getting on with our lives. Hence the request for an official hook to re-enable the unload event :-)
